frontend/src/views/web/challenge/submission.html

```diff
@@ -12,78 +12,114 @@
-                                        <input type="file" ngf-select ng-model="challenge.input_file" name="challenge.input_file" accept=".json">
-                                    </div>
-                                    <div class="file-path-wrapper">
-                                        <input class="file-path validate" type="text">
+                                <div class="row">
+                                    <div class="file-field input-field col s6" id="submission">
+                                        <div class="waves-effect waves-dark btn ev-btn-dark w-300 fs-14">
+                                            <span>Upload File</span>
+                                            <input type="file" ngf-select ng-model="challenge.input_file" name="challenge.input_file" accept=".json">
+                                        </div>
+                                        <div class="file-path-wrapper">
+                                            <input class="file-path validate" type="text">
+                                        </div>
                                     </div>
                                 </div>
-                            </div>
 
-                            <div class="row">
-                                <div class="input-field align-left col s6">
-                                    <input type="text" id="methodName" name="methodName" ng-model="challenge.methodName" required>
-                                    <span class="form-icon"><i class="fa fa-text"></i></span>
-                                    <label for="methodName">Method Name *</label>
+                                <div class="row">
+                                    <div class="input-field align-left col s6" id="submission">
+                                        <input type="text" id="methodName" name="methodName" ng-model="challenge.methodName" required>
+                                        <span class="form-icon"><i class="fa fa-text"></i></span>
+                                        <label for="methodName">Method Name *</label>
+                                    </div>
                                 </div>
-                            </div>
 
-                            <div class="row">
-                                <div class="input-field align-left col s6">
-                                    <textarea id="methodDesc" class="materialize-textarea" name="methodDesc" ng-model="challenge.methodDesc" required></textarea>
-                                    <span class="form-icon"><i class="fa fa-text"></i></span>
-                                    <label for="methodDesc">Method Description *</label>
-                                </div>                                
-                            </div>
+                                <div class="row">
+                                    <div class="input-field align-left col s6" id="submission">
```

Comment:
Also, never alter col width directly to the same element, instead use nested elements to alter widths.